src/Codec/JVM/ASM/Code/Instr.hs

```diff
@@ -32,7 +32,8 @@ data InstrState =
              , isLabelTable    :: LabelTable
              , isLastBranch    :: LastBranch
              , isRunAgain      :: Bool
-             , isNextLabel     :: Int }
+             , isNextLabel     :: Int 
+             , isLineNumber    :: Maybe Int }
```

Comment:
Mmm, i clearly have to study the lib more carefully: i mistakenly assumed Instr represented one bytecode op